secure/storage/src/in_memory.rs

```diff
@@ -76,6 +76,7 @@ impl<T: Send + Sync + TimeService> KVStorage for InMemoryStorageInternal<T> {
         Ok(())
     }
 
+    #[cfg(any(test, feature = "testing"))]
```

Comment:
you create yet another feature, I thought fuzzing == testing in the codebase

Reply:
why are we deleting the mall?

Reply:
fuzzing should be renamed to testing, since fuzzing is a subset of testing, no? let's rename them all!!!